crates/cli/src/default_scenarios/erc20.rs

```diff
@@ -80,16 +83,27 @@ impl ToTestConfig for Erc20Args {
                 from_pool: Some("admin".to_owned()),
             }]),
             setup: Some(setup_steps),
-            // transfer tokens to self
             spam: Some(vec![SpamRequest::new_tx(
                 &FunctionCallDefinition::new(token.template_name())
+                    .with_from_pool("spammers") // Senders from limited pool
                     .with_signature("transfer(address guy, uint256 wad)")
                     .with_args(&[
-                        self.token_recipient
-                            .to_owned()
-                            .unwrap_or("{_sender}".to_owned()),
-                        self.send_amount.to_string(),
-                    ]),
+                        // This will be overwritten by the fuzzer below
+                        "0x0000000000000000000000000000000000000000",
+                        &self.send_amount.to_string(),
+                    ])
+                    .with_gas_limit(55000)
+                    .with_fuzz(&[FuzzParam {
```

Comment:
this is a great idea! Only problem is that the `token_recipient` arg doesn't get used anymore. I think it would be nice to keep `token_recipient` if it's supplied, but otherwise set fuzz params. That way the default will still be to fuzz, but the user can specify a recipient if they want their tokens back for whatever reason.

Reply:
Yes good call. Updated in https://github.com/flashbots/contender/pull/366/commits/8cd02c7229a425d0cae04ba03aa989f14b2c9337